api/media_types.cc

```diff
@@ -17,6 +17,7 @@ namespace cricket {
 const char kMediaTypeVideo[] = "video";
 const char kMediaTypeAudio[] = "audio";
 const char kMediaTypeData[] = "data";
+const char kMediaTypeScreen[] = "screen";
```

Comment:
Since we no longer needs for port range allocation. Could we remove them?

Reply:
Removed.